2-Cell-Morphology-VIP.ipynb

```diff
@@ -47,14 +47,34 @@
   {
    "cell_type": "code",
    "execution_count": 2,
-   "id": "0b653310",
+   "id": "6ad3bc84",
    "metadata": {},
    "outputs": [],
    "source": [
     "platemaps = pd.read_csv('inputs/metadata/cytodata/barcode_platemap.csv')\n",
     "pmaps = pd.concat([pd.read_csv('inputs/metadata/cytodata/platemap/DOC45.46.47.48.txt', sep='\\t'),\n",
     "                   pd.read_csv('inputs/metadata/cytodata/platemap/DOC49.50.51.52.txt', sep='\\t')],\n",
-    "                 )\n",
+    "                 )"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "0836d87b",
+   "metadata": {},
+   "source": [
+    "### Filter quality control status\n",
+    "\n",
+    "All of our TP53 alleles (including our allele annotated as \"wt\") actually harbor an inactivating p53 variant. Thus, we flagged such alleles(`qc_status=False`) and discarded them in this analysis because they are not valid REFERENCES for the cmVIP framework."
```

Comment:
@shntnu do you think this is accurate and clear?

Reply:
... actually harbor an inactivating p53 mutation. Thus, we flagged such alleles(`qc_status=False`) and discarded them in this analysis because they are the wrong perturbations for the purposes of this study.